vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1022,15 +1069,80 @@ public void run() {
+//            mFloatingActionButton.setOnClickListener(new View.OnClickListener() {
+//                @Override
+//                public void onClick(View v) {
+//                    Fragment fragment = getSelectedFragment();
+//
+//                    if (!(fragment instanceof AbsHomeFragment) || !((AbsHomeFragment) fragment).onFabClick()) {
+//                        onFloatingButtonClick();
+//                    }
+//                }
+//            });
+//        }
+
+        Drawable drawable;
+
+        drawable = getResources().getDrawable(R.drawable.ic_person_black_24dp).mutate();
+        DrawableCompat.setTint(drawable, getResources().getColor(android.R.color.white));
+        mFabStartChat.setIconDrawable(drawable);
+
+        drawable = getResources().getDrawable(R.drawable.ic_add_white).mutate();
+        DrawableCompat.setTint(drawable, getResources().getColor(android.R.color.white));
+        mFabCreateRoom.setIconDrawable(drawable);
+
+        drawable = getResources().getDrawable(R.drawable.riot_tab_rooms).mutate();
+        DrawableCompat.setTint(drawable, getResources().getColor(android.R.color.white));
+        mFabJoinRoom.setIconDrawable(drawable);
+
+        // Pre-Lollipop does not support elevation so cannot have touch guard
+        // above bottomNavigationView.
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            mFloatingActionsMenu.setOnFloatingActionsMenuUpdateListener(new FloatingActionsMenu.OnFloatingActionsMenuUpdateListener() {
                 @Override
-                public void onClick(View v) {
-                    Fragment fragment = getSelectedFragment();
+                public void onMenuExpanded() {
+                    touchGuard.setAlpha(0.6f);
 
-                    if (!(fragment instanceof AbsHomeFragment) || !((AbsHomeFragment) fragment).onFabClick()) {
-                        onFloatingButtonClick();
-                    }
+                    touchGuard.setOnClickListener(new View.OnClickListener() {
+                        @Override
+                        public void onClick(View v) {
+                            mFloatingActionsMenu.collapse();
+                        }
+                    });
+                }
+
+                @Override
+                public void onMenuCollapsed() {
+                    touchGuard.setAlpha(0);
+
+                    touchGuard.setOnClickListener(null);
+                }
+            });
+        }
+
+        if (null != mFabStartChat) {
+            mFabStartChat.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    invitePeopleToNewRoom();
+                }
+            });
+        }
+        if (null != mFabCreateRoom) {
+            mFabCreateRoom.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    mFloatingActionsMenu.collapse();
```

Comment:
The two other cases does not call `collapse()`. So if it is necessary, add the 2 missing calls and if it is not, remove this one :)

Reply:
Added missing calls.